src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
```

Comment:
This file could be split up in a number of places, but it's quite convenient right now that there is a single copy command required to pull this into google3 for testing, so I'd prefer to continue growing it for now while we are under very active development